utils/ejabberd-cfg/e2etest.ini

```diff
@@ -5,6 +5,6 @@ provisioner=aioxmpp.e2etest.provision.AnonymousProvisioner
 host=localhost
 pin_store=.travis-pinstore.json
 pin_type=0
-no_verfiy=true
+no_verify=true
```

Comment:
So all of these have obviously been ineffective and also incorrect semantically; we use a pin store to avoid having to bypass verification. Would you like to add a commit which removes those?

If you do, *make sure* to include the rationale, because I don't want that to be lost.